test/commands/link.test.ts

```diff
@@ -63,4 +66,26 @@ describe('link', () => {
     const linked_components = fs.readJSONSync(linked_components_file);
     expect(linked_components).to.have.property('hello-world', component_path);
   });
+
+  it('list linked components', async () => {
+    const component_path = path.join(__dirname, '../../examples/hello-world/').replace(/\/$/gi, '').replace(/\\$/gi, '');
+
+    const log_spy = sinon.fake.returns(null);
+    sinon.replace(Link.prototype, 'log', log_spy);
+    await Link.run([component_path]);
```

Comment:
I know this test is similar to the one above, but that's pretty outdated. New tests should run the command with `.command` like this one and follow a similar format

https://github.com/architect-team/architect-cli/blob/rc/test/commands/dev.test.ts#L511